src/components/renderer/OneBlinkFormElements.tsx

```diff
@@ -184,6 +174,44 @@ function OneBlinkFormElements<T extends FormTypes._NestedElementsElement>({
 
 export default React.memo(OneBlinkFormElements)
 
+function FormElementSwitchContainer(
+  props: Omit<FormElementSwitchProps, 'isDirty' | 'setIsDirty'>,
+) {
+  const { element, formElementValidation, displayValidationMessage } = props
+  const [isDirty, setIsDirty] = useBooleanState(false)
+
+  const getValidationClass = () => {
+    if (!('elements' in element)) {
+      if (!formElementValidation) {
+        return 'ob-element__valid'
+      }
+      if (isDirty || displayValidationMessage) {
+        return 'ob-element__invalid'
+      }
+    }
+  }
+  const validationClassName = getValidationClass()
+
+  if (element.type === 'page' || element.type === 'section') {
```

Comment:
this seems wrong, should this only be returning null for `'page'` element?

Reply:
section and pages have been handled in the component above, this is really just for TS

Reply:
Yes, I meant to check that further down, to see if it had just been moved. Didn't appear so, I think this is wrong too.

Reply:
Sorry, comment posted late.

Reply:
Could we type the element as excluding those 2 instead?